src/features/projectsV2/ProjectDetails/components/microComponents/SingleCarouselImage.tsx

```diff
@@ -1,44 +1,37 @@
-import { useMemo } from 'react';
 import { SliderImage } from '../../../../projects/components/PlantLocation/ImageSlider';
 import styles from '../../styles/Slider.module.scss';
```

Comment:
Avoid using code from the old version of projects. Set up the SliderImage type within v2 itself.